apps/pie-storybook/stories/pie-card-container.stories.ts

```diff
@@ -131,8 +132,8 @@ const Template: TemplateFunction<CardContainerProps> = ({
             ?disabled="${disabled}"
             .aria="${aria}"
             ?isDraggable="${isDraggable}">
-            ${unsafeStatic(slot)}
-        </pie-card-container>
+                ${unsafeStatic(DOMPurify.sanitize(slot))}
```

Comment:
would it be possible for us to create a storybook helper function that combines these calls? Then we can standardise using that everywhere for slots rather than remembering to manually sanitize